### PR TITLE
Turn on/off screen by adbe screen command

### PR DESCRIPTION
**Pull request by STFBEE**
Screen toggle feature for #76 

Tested on Android 11 @ Pixel 4 and Android 6.0.1 @ Nexus 7 2013

### CODE REVIEW

File: adbe/adb_enhanced.py [hunk 1]
```diff
@@ -50,6 +50,9 @@
 # Value to be return as 'unknown' to the user
 _USER_PRINT_VALUE_UNKNOWN = 'unknown'
 
+SCREEN_ON = 1
```
Comment (ashishb) [inline adbe/adb_enhanced.py:53]: Nit. Consider prefixing them with "_" to make them private.

File: adbe/adb_enhanced.py [hunk 2]
```diff
@@ -1531,3 +1534,25 @@ def _error_if_min_version_less_than(min_acceptable_version, device_serial=None):
 def _is_emulator():
     qemu = get_adb_shell_property('ro.kernel.qemu')
     return qemu is not None and qemu.strip() == '1'
+
+
+def switch_screen(switch_type):
+    if switch_type == SCREEN_TOGGLE:
+        c, o, e = toggle_screen()
+    else:
```
Comment (ashishb) [inline adbe/adb_enhanced.py:1542]: Nit: You might make an early exit here by 

```
if switch_type == SCREEN_TOGGLE:
        c, o, e = toggle_screen()
        return o
```

And then the `else` case won't require an indentation.

**Review by ashishb — APPROVED** [review]
Thanks for your contribution. I am approving the change but consider addressing some minor comments that I added.

Optional: Consider adding a basic test to https://github.com/ashishb/adb-enhanced/blob/master/tests/adbe_tests.py 
GitHub actions are configured to run on 5+ Android API versions simultaneously. So, we will see the test results on all of them.

File: adbe/adb_enhanced.py (hunk 1 above)
Comment (STFBEE) [reply adbe/adb_enhanced.py:53]: But it's used in `main.py`
Should i extract it into another file maybe?

File: adbe/adb_enhanced.py (hunk 2 above)
Comment (STFBEE) [reply adbe/adb_enhanced.py]: I lost an error message here, i was fixed in new commit

File: adbe/adb_enhanced.py (hunk 1 above)
Comment (ashishb) [reply adbe/adb_enhanced.py:53]: My bad. Leave it as-is then.

**Comment by ashishb** [comment]
I fixed the minor lint issues along-side another issue in 1aedc71868a1ad5f4b2fd05201a1a8507e38b91a

**Pull request merged**